code/src/CharachteristicsButtonOption.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const CharacteristicsButtonOptions = ({
+    characteristicsName, 
+    setInputCharacteristics, 
+}) => {
+
+const onInputCharacteristicsChanged = (e) => {
+    setInputCharacteristics(e.target.value);
+    console.log(e.target.value)
+}
+    return (
+        <>
+            <label htmlFor={characteristicsName}>{characteristicsName}</label>
+                <input 
+                    name="characteristics"
+                    id={characteristicsName} 
+                    value={characteristicsName}
+                    type="radio"
+                    onChange={(e) => {
+                        onInputCharacteristicsChanged(e);
```

Comment:
Really nice that you're dividing the input on separate lines, since it makes code easier to read and follows best practice.